mailbox/api/src/test/java/org/apache/james/mailbox/events/ErrorHandlingContract.java

```diff
@@ -121,32 +122,50 @@ default void listenerShouldNotReceiveWhenFailsGreaterThanMaxRetries() {
         eventBus().register(eventCollector, new EventBusTestFixture.GroupA());
         eventBus().dispatch(EVENT, NO_KEYS).block();
 
+        TimeUnit.SECONDS.sleep(1);
         assertThat(eventCollector.getEvents())
             .isEmpty();
     }
 
     @Test
-    default void retriesBackOffShouldDelayByExponentialGrowth() {
+    default void exceedingMaxRetriesShouldStopConsumingFailedEvent() throws Exception {
+        ThrowingListener throwingListener = throwingListener();
+
+        eventBus().register(throwingListener, new EventBusTestFixture.GroupA());
+        eventBus().dispatch(EVENT, NO_KEYS).block();
+
+        TimeUnit.SECONDS.sleep(5);
```

Comment:
Why do you need a sleep after a block?

Reply:
I need to sleep to make sure my getting size of listener at the statement in bellow line is the final result. If I get it immediately - result is 0 but a second latter could be 1 because retrying process is just succeeded.

Reply:
slpeep(5) what? 5 ms? could you develop?

Also 5 ms seems low, let's go for 100 ms to get a good stability.